programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -155,16 +155,20 @@ static uv_cond_t stop_cond;
 IMPL_ENUM(event, EVENT_ACTIONS)
 
 #if _WIN32
-static char sockfile[] = "\\\\.\\pipe\\ziti-edge-tunnel.sock";
-static char eventsockfile[] = "\\\\.\\pipe\\ziti-edge-tunnel-event.sock";
+static char SOCKET_PATH[] = "\\\\.\\pipe\\";
+static char sockfilebase[] = "ziti-edge-tunnel.sock";
+static char eventsockfilebase[] = "ziti-edge-tunnel-event.sock";
 #elif __unix__ || unix || ( __APPLE__ && __MACH__ )
 #include <grp.h>
 #include <sys/un.h>
-#define SOCKET_PATH "/tmp/.ziti"
```

Comment:
`SOCKET_PATH` is used in at least one `#ifdef`. Changing it from a macro to a variable inadvertently eviscerates `make_socket_path()`